lms/templates/admin/organization.html.jinja2

```diff
@@ -0,0 +1,38 @@
+{% import "macros.jinja" as macros %}
+
+{% extends "lms:templates/admin/base.html.jinja2" %}
+{% block header %}
+    Organization {{org.id}}
+{% endblock %}
+
+{% block content %}
+    <fieldset class="box">
+    <legend class="label has-text-centered">Organization</legend>
+    <form method="POST" action="{{ request.route_url("admin.organization", id_=org.id) }}">
+        <input type="hidden" name="csrf_token" value="{{get_csrf_token()}}">
+        {{ macros.disabled_text_field("ID", org.public_id(request.region)) }}
+        {{ macros.form_text_field(request, "Name", "name", org.name) }}
+        {% call macros.field_body("Enabled") %}
+            <label class="checkbox">
+                <input {% if org.enabled %}checked{% endif%} type="checkbox"
+                    name="enabled" disabled>
+            </label>
+        {% endcall %}
+        {{ macros.created_updated_fields(org) }}
+        <div class="has-text-right mb-6">
+            <input type="submit" class="button is-primary" value="Save" />
+        </div>
+    </form>
+
+    <legend class="label has-text-centered">Organization application instances</legend>
+        {% if org.application_instances %}
+            {{ macros.object_list_table(
+                    request, 'admin.instance.id', org.application_instances,
+                    ['Consumer key', 'Deployment ID', 'Tool name', 'URL'],
+                    ['consumer_key', 'deployment_id', 'tool_consumer_instance_name', 'lms_url']
```

Comment:
I think `requesters_email` would be very good here, as it's never blank, and contains a domain.

If we can fix the scrolling issues we can probably have them all. If not, try and work out which of `consumer_key` or `deployment_id` is less useful in this context?